src/app/main.cpp

```diff
@@ -178,6 +178,8 @@ int main(int argc, char *argv[])
                                  .arg(QLatin1String("-h (or --help)")));
         }
 
+        app->initInstance();
```

Comment:
I really don't like splitting its initialization. Recently (for quite a while now, actually) I'm thinking of doing a deeper rework of the initialization code, but I don't have time for that yet. So if others decide that this improvement is really important, I will approve this method as a workaround.